python/tank/descriptor/descriptor_config.py

```diff
@@ -226,36 +226,34 @@ def _find_interpreter_location(self, path):
                 "No interpreter file for the current platform found at '%s'." % interpreter_config_file
             )
 
-    def _get_roots_data(self):
+    @property
```

Comment:
nitpick: when diff screws up like here, I generally reorder the function defition order so that there is less noise in the PR.